Assets/MixedRealityToolkit.SDK/StandardAssets/Materials/MRTK_Standard_GlowingCyan.mat

```diff
@@ -9,8 +9,8 @@ Material:
   m_PrefabAsset: {fileID: 0}
   m_Name: MRTK_Standard_GlowingCyan
   m_Shader: {fileID: 4800000, guid: 5bdea20278144b11916d77503ba1467a, type: 3}
-  m_ShaderKeywords: _BORDER_LIGHT_USES_HOVER_COLOR _DISABLE_ALBEDO_MAP _HOVER_LIGHT
-    _INNER_GLOW
+  m_ShaderKeywords: _BORDER_LIGHT _BORDER_LIGHT_USES_HOVER_COLOR _DISABLE_ALBEDO_MAP
```

Comment:
Is this material change unique to HandInteractionExamples or good for all uses of this standard asset?

Reply:
This was used for the large gem in the Touch interaction example. The change is good for other examples if it is used.